### PR TITLE
[Console] Fix Application::setDefaultCommand() missing return in docblock

### PR DESCRIPTION
**Pull request by ogizanagi**
| Q | A |
| --- | --- |
| Branch? | master |
| License | MIT |

Despite the fact IDEs are clever enough to make this useless, I think it's clearly part of the contract of this public method anyway 😄

### CODE REVIEW

**Comment by fabpot** [comment]
Instead, I would remove the `return $this;`, which is not present in previous versions and not used anywhere else (except for a deprecated method) in this class. What do you think?

**Comment by HeahDude** [comment]
@fabpot see the description of https://github.com/symfony/symfony/pull/19748 :)

File: src/Symfony/Component/Console/Application.php [hunk 1]
```diff
@@ -1002,6 +1002,8 @@ private function findAlternatives($name, $collection)
      *
      * @param string $commandName     The Command name
      * @param bool   $isSingleCommand Set to true if there is only one command in this application
+     *
+     * @return Application The current application
```
Comment (HeahDude) [inline src/Symfony/Component/Console/Application.php]: can be `self` without extra comment since it's not any implementation but the "current"

**Comment by ogizanagi** [comment]
@fabpot : I personally don't mind, both are fine to me. 
As reported by @HeahDude, the fluent interface here was intended to ease creating an application in a minimum amount of code. But as we don't use it in other methods, it does not make entirely sense, indeed.

**Comment by chalasr** [comment]
To give an opinion, even if I'm globally against fluent interfaces, I think this one makes sense.
I often write single-command apps and I really like the fact that the app is created and ran without need of any assignment, like in the example given in #19748 description. Most often, it represents the only logic in the whole executable

**Comment by fabpot** [comment]
Thank you @ogizanagi.

**Pull request merged**